certified-connectors/Africa's Talking SMS/apiDefinition.swagger.json

```diff
@@ -210,6 +210,265 @@
+            "description": "Content-type",
+            "type": "string"
+          }
+        },
+        "description": "A new delivery report for a message"
+      },
+      "post": {
+        "responses": {
+          "200": {
+            "description": "200"
+          }
+        },
+        "summary": "When you receive a Delivery Report for a sent message",
+        "description": "Triggered when you receive a Delivery Report for a message initially sent",
+        "operationId": "DlrNotification",
+        "x-ms-visibility": "important",
+        "x-ms-trigger": "single",
+        "parameters": [
+          {
+            "name": "username",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-visibility": "important",
+            "x-ms-url-encoding": "single",
+            "x-ms-summary": "Username",
+            "description": "Your application's username."
+          },
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "url": {
+                  "type": "string",
+                  "description": "url",
+                  "x-ms-visibility": "internal",
+                  "title": "",
+                  "x-ms-notification-url": true
+                }
+              },
+              "x-ms-visibility": "internal",
+              "required": [
+                "url"
+              ]
+            },
+            "x-ms-visibility": "internal"
+          }
+        ]
+      }
+    },
+    "/api/v1/powerapps/{username}/sms/inbox/callback": {
+      "x-ms-notification-content": {
+        "schema": {
+          "type": "object",
+          "properties": {
+            "date": {
+              "type": "string",
+              "description": "date"
+            },
+            "from": {
+              "type": "string",
+              "description": "from"
+            },
+            "id": {
+              "type": "string",
+              "description": "id"
+            },
+            "linkId": {
+              "type": "string",
+              "description": "linkId"
+            },
+            "text": {
+              "type": "string",
+              "description": "text"
+            },
+            "to": {
+              "type": "string",
+              "description": "to"
+            },
+            "networkCode": {
+              "type": "string",
+              "description": "networkCode"
+            }
+          }
+        },
+        "description": "A new message in your inbox"
+      },
+      "post": {
+        "responses": {
+          "200": {
+            "description": "200"
```

Comment:
@crispinkoech Can you please provide a valid description for the "200" response code instead of "200"